orchestration/flows/bl832/tomography_hpc.py

```diff
@@ -0,0 +1,343 @@
+from abc import ABC, abstractmethod
+from dotenv import load_dotenv
+import logging
+import os
+from pathlib import Path
+from typing import Optional, Dict, Any
+
+from orchestration.flows.bl832.config import Config832
+from orchestration.nersc import NerscClient
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class TomographyHPCController(ABC):
+    """
+    Abstract class for tomography HPC controllers.
+    Provides interface methods for reconstruction and building multi-resolution datasets.
+
+    Args:
+        ABC: Abstract Base Class
+    """
+    def __init__(self):
+        pass
+
+    @abstractmethod
+    def reconstruct(
+        self,
+        file_path: str = "",
+        is_export_control: bool = False,
```

Comment:
I think it's unlikely that these classes will need to know about export control. Whatever calls them will do a lot of things like not copying files before deciding not to reconstruct them.